Clock.py

```diff
@@ -127,21 +126,29 @@ def announce(s, kob, sender):
     from pykob import config, kob, morse, log
     from distutils.util import strtobool
 
-    log.log("Starting Clock")
-
-    clock_parser = argparse.ArgumentParser(parents=[config.PortOverride, config.SoundtOverride, config.WPMOverride])
-    clock_parser.add_argument("-b", "--begin", default=900, type=int, help="Beginning of time announcements ", metavar="time", dest="Begin")
-    clock_parser.add_argument("-e", "--end", default=2200, type=int, help="End of time announcements ", metavar="time", dest="End")
-    clock_parser.add_argument("-i", "--interval", default=60, type=int, help="The time announcement interval in minutes", metavar="minutes", dest="Interval")
-    clock_parser.add_argument("-t", "--text", action='store_true', default=False, help="Whether to print text locally as it's sent to the sounder", dest="Text")
-    args = clock_parser.parse_args()
+    #log.log("Starting Clock")
+
+    arg_parser = argparse.ArgumentParser(description="Morse Cuckoo Clock", parents=\
+     [\
+      config.serial_port_override, \
+      config.code_type_override, \
+      config.sound_override, \
+      config.sounder_override, \
+      config.spacing_override, \
+      config.min_char_speed_override, \
+      config.text_speed_override])
+    arg_parser.add_argument("-B", "--begin", default=900, type=int, help="Beginning of time announcements ", metavar="time", dest="Begin")
```

Comment:
Why switch to all-uppercase characters for options?  Especially when most of the command line is lower case commands (e.g. 'python3 Clock.py') is there anything wrong with using lowercase option characters, which in my experience are more common in UNIX/Linux commands?

Reply:
I will switch Clock back to lowercase option characters.